frontend/src/components/concertPlaylist.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
```

Comment:
What is this page for? Is this page for the work that Miguel is doing?

Reply:
Yes this is the page Miguel working on.

Reply:
Ok can we remove the file from this PR since it should go in miguels PR

Reply:
I removed it.